pulp_2to3_migration/app/plugin/rpm/migrator.py

```diff
@@ -271,11 +271,15 @@ def relate_packages_to_group(self, group_dc):
         packages = group_dc.content.packages
         package_list = [pkg['name'] for pkg in packages]
         pulp2_repo_id = group_dc.extra_data.get('pulp2_repo_id')
-        pulp2_repo = Pulp2Repository.objects.get(pulp2_repo_id=pulp2_repo_id)
-        # all pulp2 unit_ids for rpm within the pulp2repo
-        unit_ids = Pulp2RepoContent.objects.filter(
-            pulp2_repository=pulp2_repo,
-            pulp2_content_type_id='rpm').values_list('pulp2_unit_id', flat=True).iterator()
+        pulp2_repo = Pulp2Repository.objects.filter(pulp2_repo_id=pulp2_repo_id).first()
```

Comment:
@goosemania  i debugged the 'variable referenced before assignment' error. This is the fix, problem : we rely that pulp2 is migrated, this would not work if, for example, we'd like to migrate content only. I think this issue might also be present in the errata migration.

Reply:
Yeah, we need to rule those out at early stages, any content that needs a repo content data, should be skipped at migration time. Will you file an issue?

FWIW, Katello is fine, they always migrate everything for a plugin.

Reply:
I didn't realize you are out today, so I filed a one https://pulp.plan.io/issues/6511